servicex/templates/flask_app-configmap.yaml

```diff
@@ -18,12 +18,16 @@ data:
     # Enable JWT auth on public endpoints
     ENABLE_AUTH={{- ternary "True" "False" .Values.app.auth }}
 
+    # Globus configuration
+    GLOBUS_CLIENT_ID = '{{ .Values.app.globusClientID }}'
+    GLOBUS_CLIENT_SECRET = '{{ .Values.app.globusClientSecret }}'
```

Comment:
There should be placeholders for these in `values.yaml`  - many people just go to this file to pick up the values to set